pkg/virtctl/root.go

```diff
@@ -100,7 +101,7 @@ func NewVirtctlCommand() (*cobra.Command, clientcmd.ClientConfig) {
 		vm.NewExpandCommand(clientConfig),
 		memorydump.NewMemoryDumpCommand(clientConfig),
 		pause.NewPauseCommand(clientConfig),
-		pause.NewUnpauseCommand(clientConfig),
+		unpause.NewUnpauseCommand(clientConfig),
```

Comment:
```suggestion
		unpause.NewCommand(clientConfig),
```

No need to repeat ourselves.